app/models/api/fake_data/hses.rb

```diff
@@ -0,0 +1,25 @@
+require "fake_api_response_wrapper"
+require "fake_issues"
+class Api::FakeData::Hses
+  class Issue
+    include FakeApiResponseWrapper
+    attr_accessor :id
+
+    def initialize(options_arr)
+      options = Hash[*options_arr]
```

Comment:
i don't love this, but couldn't figure out a better way to optionally pass in arguments (the double splat operator would require creating an initializer for Issues, which also didn't feel great.

very open to different ways of handling the optional arguments being passed into our Api wrapper!

Reply:
[suggestion] how about

```
def initialize(options = {})
  @id = options[:id]
end
```

I think passing an options hash around explicitly is a bit cleaner, though I guess you'd also need an initializer for Issues. I think I prefer that form, and there could be a common base class that provides the no-op initializer for all of the FakeData classes.

Reply:
it's definitely cleaner!

i was thinking about that no-op initializer! for the moment i could just put it in `Hses` maybe?